proposals/artifacts.md

```diff
@@ -0,0 +1,58 @@
+# OCI artifact project proposal
+
+## Abstract
+Container registries, implementing the [distribution-spec][distribution-spec], provide reliable, highly scalable, secured storage services for container images. Customers either use a cloud provider implementation, vendor implementations, or instance the open source implementation of distribution. They configure security and networking to assure the images in the registry are locked down and accessible by the resources required. Cloud providers and vendors often provide additional values atop their registry implementations from security to productivity features. 
```

Comment:
/s/Customers either use a cloud provider implementation, vendor implementations, or instance the open source implementation of distribution./Customers use cloud provider implementations, vendor implementations, and instances of the open source implementation of docker distribution./

Reply:
/s/values/value/